src/gatling/conf/gatling.conf

```diff
@@ -10,11 +10,12 @@ gatling {
     #runDescription = ""          # The description for this simulation run, displayed in each report
     #encoding = "utf-8"           # Encoding to use throughout Gatling for file and string manipulation
     #simulationClass = ""         # The FQCN of the simulation to run (when used in conjunction with noReports, the simulation for which assertions will be validated)
-    #mute = false                 # When set to true, don't ask for simulation name nor run description (currently only used by Gatling SBT plugin)
-    #elFileBodiesCacheMaxCapacity = 200        # Cache size for request body EL templates, set to 0 to disable
-    #rawFileBodiesCacheMaxCapacity = 200       # Cache size for request body Raw templates, set to 0 to disable
-    #rawFileBodiesInMemoryMaxSize = 1000       # Below this limit, raw file bodies will be cached in memory
-
+    #elFileBodiesCacheMaxCapacity = 200      # Cache size for request body EL templates, set to 0 to disable
```

Comment:
This doesn't look to be the latest version of the file. Please copy from the template-performance repo.

Reply:
Copied and pasted from template